http/README.md

```diff
@@ -93,7 +92,7 @@ channels:
                 Content-Type:
                   type: string
                   enum: ['application/json']
-            bindingVersion: '0.1.0'
+            bindingVersion: '0.2.0'
 ```
 
-[schemaObject]: https://www.asyncapi.com/docs/specifications/2.0.0/#schemaObject
\ No newline at end of file
+[schemaObject]: https://www.asyncapi.com/docs/specifications/latest/#schemaObject
```

Comment:
Just a thought. Would be cool to have a redirect on the website that would let us point to the latest minor version of a major. For instance, something like:

```suggestion
[schemaObject]: https://www.asyncapi.com/docs/specifications/3/#schemaObject
```

Which would redirect to https://www.asyncapi.com/docs/specifications/3.0.0/#schemaObject for now but would resolve to https://www.asyncapi.com/docs/specifications/3.4.0/#schemaObject in the future (when 3.4.0 is out).

Reply:
If this is not implemented yet, I would love to have this in place. For sure super useful when sharing links to the ref for a particular major version.